examples/loading_spinners/src/circular.rs

```diff
@@ -0,0 +1,470 @@
+//! Show a circular progress indicator.
+use iced::advanced::layout;
+use iced::advanced::renderer;
+use iced::advanced::widget::tree::{self, Tree};
+use iced::advanced::{Clipboard, Layout, Renderer, Shell, Widget};
+use iced::event;
+use iced::time::Instant;
+use iced::widget::canvas::{self, Cursor, Program};
+use iced::window::{self, RedrawRequest};
+use iced::{Background, Color, Element, Rectangle};
+use iced::{Event, Length, Point, Size, Vector};
+
+use super::easing::{self, Easing};
+
+use std::f32::consts::PI;
+use std::time::Duration;
+
+type R<Theme> = iced::Renderer<Theme>;
+
+const MIN_RADIANS: f32 = PI / 8.0;
+const WRAP_RADIANS: f32 = 2.0 * PI - PI / 4.0;
+const BASE_ROTATION_SPEED: u32 = u32::MAX / 80;
+
+#[allow(missing_debug_implementations)]
+pub struct Circular<'a, Theme>
+where
+    Theme: StyleSheet,
+{
+    size: f32,
+    bar_height: f32,
+    style: <Theme as StyleSheet>::Style,
+    easing: &'a Easing,
+    cycle_duration: Duration,
+    rotaion_duration: Duration,
```

Comment:
There is an error here `rotaion` should be `rotation`

Reply:
Oops, fixed it in 9d69af10ccb4317f01bff32e2e1e58d498b2fc89